src/state/queries/profile.ts

```diff
@@ -191,68 +189,22 @@ export function useProfileFollowMutationQueue(
 }
 
 function useProfileFollowMutation() {
-  return useMutation<
-    {uri: string; cid: string},
-    Error,
-    {did: string; skipOptimistic?: boolean}
-  >({
+  return useMutation<{uri: string; cid: string}, Error, {did: string}>({
     mutationFn: async ({did}) => {
       return await getAgent().follow(did)
     },
-    onMutate(variables) {
-      if (!variables.skipOptimistic) {
-        // optimistically update
-        updateProfileShadow(variables.did, {
-          followingUri: 'pending',
-        })
-      }
-    },
     onSuccess(data, variables) {
-      if (!variables.skipOptimistic) {
-        // finalize
-        updateProfileShadow(variables.did, {
-          followingUri: data.uri,
-        })
-        track('Profile:Follow', {username: variables.did})
```

Comment:
Technically we accidentally were missing this event entirely because it was in a falsy branch. So moving it out is kind of a bugfix. I don't see it emitted elsewhere.